src/libs/error.js

```diff
@@ -17,7 +17,8 @@ export const withErrorReporting = _.curry((title, fn) => async (...args) => {
   try {
     return await fn(...args)
   } catch (error) {
-    reportError(title, error)
+    await reportError(title, error)
+    throw error
```

Comment:
This will have an effect everywhere we use `withErrorReporting`. Would you mind removing the `throw`.
In an earlier revision, I saw you had some retry logic - was this change related to that?

Reply:
Adding the throw is how to make `withErrorReporing` preserve the failed state of promises

Reply:
To prevent unexpected errors from occurring around the site, how would you feel about not using `withErrorReporting` in `reloadBillingProject` and handling that imperatively with a `try/catch/throw`? 

Or adding a new function called `withPreservedErrorReporting` (or a better name...). This will give us control in introducing the new wrapper where appropriate